civihr_default_theme/scss/components/_ui-select.scss

```diff
@@ -76,6 +76,17 @@
 * TEMP fixes till ui-select styling is fixed
 * Dropdown ul opacity was set to 0
 */
-.chr_leave-request-modal__form .ui-select-container ul {
-  opacity: 1 !important;
+.chr_leave-request-modal__form {
```

Comment:
Sorry did not notice this before, but `_ui-select.scss`should contain only the global styles. This should go to `_leave-request-modal.scss`